crates/astria-sequencer/Cargo.toml

```diff
@@ -45,6 +50,7 @@ divan = { workspace = true, optional = true }
 futures = { workspace = true }
 hex = { workspace = true, features = ["serde"] }
 ibc-types = { workspace = true, features = ["with_serde"] }
+itertools = { workspace = true }
```

Comment:
The instance where this was used allocates. I'd like to not allocate. :)

Reply:
Removed in [44170ce](https://github.com/astriaorg/astria/pull/1492/commits/44170ceefc479c5f7839a388b322ac2412962df2).